tasties_app/tests/test_recipes_view.py

```diff
@@ -34,6 +36,20 @@ def test_view_recipe_not_found(self, client, signed_up_credentials, login_to_sit
         assert response.status_code == 302
         assert response.url == '/'
 
+    def test_add_comment_function(self, client, comment, recipe_test, create_user):
+        client.force_login(create_user)
+        response = client.get(f'/view_recipe/{recipe_test.id}/?comment-adding={comment.comment_text}')
+        assert response.status_code == 200
+        comment_test = Comment.objects.get(author_id=comment.author_id, comment_text=comment.comment_text)
+        assert comment_test.comment_text == comment.comment_text
+
+    def test_add_comment_function_invalid(self, client, create_user, recipe_test, comment):
```

Comment:
You're taking a `comment` parameter here, I don't see a fixture by that name anywhere. What is this parameter? Where did it come from?

Reply:
This is the first fixture in the conftest file 
![image](https://user-images.githubusercontent.com/116442298/213177939-51b8e7e1-fb43-4157-bd74-a63d4f6f8b7c.png)